variables.tf

```diff
@@ -36,6 +36,12 @@ variable "condition" {
   default     = null
 }
 
+variable "policy_bindings" {
+  description = "(Optional) A list of IAM policy bindings."
+  type        = any
+  default     = null
+}
+
```

Comment:
i assume this makes role optional... but it is still required...please check in all modules...